event_routing_backends/tests/test_models.py

```diff
@@ -76,8 +77,26 @@ def test_allowed_hosts(self):
         router = RouterConfigurationFactory(
             configurations=config_fixture,
             enabled=True,
+            route_url='http://test3.com',
             backend_name='first'
         )
 
         hosts = router.get_allowed_hosts(original_event)
         self.assertEqual(config_fixture[:1], hosts)
+
+    def test_model_cache(self):
```

Comment:
Can we modify this test to create 2 router configurations and them match then to assert `get_enabled_routers` can return more than one enteries.